uxarray/core/dataarray.py

```diff
@@ -429,6 +434,78 @@ def integrate(
 
         return uxda
 
+    def zonal_mean(self, lat=(-90, 90, 10)):
```

Comment:
I think it'd be nice to get the API right, i.e. have some optional variables as such as `method` that represents `weights candidate faces by the length of intersection by a given line of constant latitude.` also the choice of conservative or non-conservative etc. This will make it easier for new additions to this work.

Reply:
Think we should call the method `non_conservative_zonal_mean` then, keeping the current parameters. Thoughts?

Reply:
> Think we should call the method `non_conservative_zonal_mean` then, keeping the current parameters. Thoughts?

I like the current neat small function signature. How about keep it as is and have optional `conservative` `True/False`

Reply:
That works with me. However, I think adding the `conservative` parameter might make more sense once we implement the conservative method, not to confuse the user.

Reply:
ok, fix this one and we can merge it then